pkg/networkservice/common/refresh/client.go

```diff
@@ -29,17 +29,18 @@ import (
 	"github.com/pkg/errors"
 	"google.golang.org/grpc"
 
+	"github.com/networkservicemesh/sdk/pkg/networkservice/common/begin"
+	"github.com/networkservicemesh/sdk/pkg/networkservice/utils/metadata"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+
 	"github.com/networkservicemesh/api/pkg/api/networkservice"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
 	"github.com/networkservicemesh/sdk/pkg/tools/clock"
-	"github.com/networkservicemesh/sdk/pkg/tools/extend"
-	"github.com/networkservicemesh/sdk/pkg/tools/serializectx"
 )
 
 type refreshClient struct {
 	chainCtx context.Context
```

Comment:
I had kept them because we are emitting goroutines, and without chainctx we can slightly leak them on application close (which complicates testing for other kinds of go routine leaks.